### PR TITLE
Navigation setup with react-router and express server

### PR DESCRIPTION
**Pull request by GiuliaTeggi**
I installed the react-router-dom package which is needed to navigate between different components/pages, added the router to app.js and links to the navbar using the react-router-dom built-in `<Link />` component. Now all links in the navbar are working and rendering the correct react components. :smiley: 
 
Then I made sure that the express server was recognising the different react routes (if you typed directly in the browser localhost:3000/schedule before this step, you would get a 404 error from the server in the console).  
This is done by adding `app.get('*', (req, res) => res.sendFile(path.join(__dirname, '..', 'dist/index.html')))` at the end of src/index.js (that is, after all of the other requests are made e.g. to the database or to external apis) - which basicaly tells the server to respond to all (`*`) get requests with the index.html located in dist (the compiled index.html), which is the 'entry point' to our react components.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "name": "court-watch",
   "version": "1.0.0",
-  "description":
-    "A notifications, bookings and data gathering app for court observers at hearings where defendants are women.",
+  "description": "A notifications, bookings and data gathering app for court observers at hearings where defendants are women.",
```
Comment (GiuliaTeggi) [inline package.json:4]: I didn't change anything on package.json (other than installing react-router-dom). This difference is probably due to the fact that we have different spacing settings in our editors :wink:

**Review by haydnba — APPROVED** [review]
Amazing work 💯

File: package.json (hunk 1 above)
Comment (haydnba) [reply package.json:4]: yeah we should sort that out tomorrow!

File: src/app.js [hunk 2]
```diff
@@ -4,4 +4,8 @@ const app = express();
 
 app.use(express.static(path.join(__dirname, '..', 'dist'))); 
 
+// -- all database and external requests will go here --
+
+app.get('*', (req, res) => res.sendFile(path.join(__dirname, '..', 'dist/index.html')));
```
Comment (haydnba) [inline src/app.js:9]: Interested to see how we handle post requests with the react-router...

**Pull request merged**